[Challenge 42] Spring boot actuator hiding api key (#1107)
OWASP/wrongsecrets: src/main/resources/explanations/challenge42_reason.adoc

*Why hiding API key in the audit events is not a good idea?*

> In a secure and well-architected system, it's crucial to protect sensitive information such as API keys.
> Audit events play a significant role in tracking and monitoring activities within an application.
> 
> The API keys should not be visible in plain text in the logs or any other output.
> 
> You should always ensure that API keys are not exposed in clear text within the audit logs.

Reviewer: ```suggestion
In a secure and well-architected system, protecting sensitive information such as API keys is crucial.
Audit events are significant in tracking and monitoring activities within an application.

Ensure that API keys are not visible in plain text in the (audit) logs or any other output.
```
